src/main/resources/application.yml

```diff
@@ -6,7 +6,7 @@ server:
 spring:
   datasource:
     password:
-    url: jdbc:postgresql://localhost:5432/registration
+    url: jdbc:postgresql://localhost:5430/registration
```

Comment:
Please change the postgresql port according to your setup. For me it's 5430 where my postgres returning response.